rustls-platform-verifier/src/verification/android.rs

```diff
@@ -305,4 +284,38 @@ impl ServerCertVerifier for Verifier {
             }
         }
     }
+
+    fn verify_tls12_signature(
+        &self,
+        message: &[u8],
+        cert: &pki_types::CertificateDer<'_>,
+        dss: &DigitallySignedStruct,
+    ) -> Result<HandshakeSignatureValid, TlsError> {
+        verify_tls12_signature(
+            message,
+            cert,
+            dss,
+            &rustls::crypto::ring::default_provider().signature_verification_algorithms,
+        )
+    }
+
+    fn verify_tls13_signature(
+        &self,
+        message: &[u8],
+        cert: &pki_types::CertificateDer<'_>,
+        dss: &DigitallySignedStruct,
+    ) -> Result<HandshakeSignatureValid, TlsError> {
+        verify_tls13_signature(
+            message,
+            cert,
+            dss,
+            &rustls::crypto::ring::default_provider().signature_verification_algorithms,
+        )
+    }
+
+    fn supported_verify_schemes(&self) -> Vec<SignatureScheme> {
```

Comment:
Out of curiosity, howcome supported schemes uses a `Vec` instead of `&'static []`? Is it expected these could change at runtime somehow inside a provider?

Reply:
I'm not sure. The return type for this fn has remained unchanged since it was introduced ~4 years ago: https://github.com/rustls/rustls/commit/1e98fa2eb1004b0ddbf68bdbcbf960891667e10b

Reply:
Thanks for the extra context, I didn't realize this wasn't a new introduction with `0.22`. I think that makes sense for the most part, so I'll leave it be 👍